docs/topics/BrowserPlugin.adoc

```diff
@@ -66,9 +66,10 @@ The KeePassXC-Browser extension lets you automatically populate the entries from
 
 2. Open your web browser. The KeePassXC-Browser extension icon in your browser window will change based on its connection state. The figure below shows the different states.
 +
-*(A)* KeePassXC is not running or is disconnected +
-*(B)* Connected to KeePassXC, but database is locked +
-*\(C)* Connected to KeePassXC and ready to use
+*(A)* KeePassXC is not running or is disconnected. +
+*(B)* KeePassXC is running but current database is not connected to it. +
+*\(C)* Connected to KeePassXC, but database is locked. +
+*(D)* Connected to KeePassXC and ready to use. If the icon is shown with a number, it indicates the number of credentials found for the current site.

```

Comment:
We interchange the term disconnected with the app and the database. Let's be clearer.
```
(A) The extension cannot communicate with KeePassXC
(B) The extension can communicate, but the current database is not connected
(C) The extension can communicate, but the current database is locked
(D) The extension is ready to be used and the number of credentials found for the current site will be shown.
```

Can we include a shot with the number of credentials shown?

Reply:
Something like this? The icon size is a bit different from others:
<img width="259" alt="Screenshot 2024-06-09 at 15 45 28" src="https://github.com/keepassxreboot/keepassxc/assets/24570482/1f92ccfc-75c8-4d9d-b6c7-4c5a8fdcb20a">

Reply:
Hmm, you might have to just doctor up a sample icon with number instead of the smaller one. I was thinking both would just be labeled with one (D).

Reply:
Suggestion:
<img width="516" alt="Screenshot 2024-06-09 at 21 06 12" src="https://github.com/keepassxreboot/keepassxc/assets/24570482/a45a9945-97b9-4342-a650-8f288d1feb32">

Reply:
That's great